sql/core/src/test/resources/sql-tests/analyzer-results/percentiles.sql.out

```diff
@@ -149,6 +149,226 @@ Aggregate [median(v#x) AS median(v)#x, percentile(v#x, cast(0.5 as double), 1, 0
                +- LocalRelation [k#x, v#x]
 
 
+-- !query
+SELECT
+  round(v, 0) WITHIN GROUP (ORDER BY v)
+FROM aggr
+-- !query analysis
+org.apache.spark.sql.AnalysisException
+{
+  "errorClass" : "_LEGACY_ERROR_TEMP_1023",
```

Comment:
since we are here, shall we create error classes for them?

Reply:
Let me make a separate PR for this, because this mistake is very common and can be easily corrected. It's not easy to review, and I'm also not good at resolving conflicts.